docs/installation/docker-custom.md

```diff
@@ -1,38 +1,85 @@
-store passwords as secrets and set up TLS.
+If you are deploying Islandora on a server that is publicly accessible, there are extra precautions you should take for
+performance and security reasons.
 
-## Getting Started
+For local development we bind mount our codebase (Drupal) folder so that we can do development work locally, outside of 
+our Docker container. For a production site we don’t want to do this. Instead, we make changes to our development site, 
+then build a Drupal image with our changes that we can use on our production site instead of the default Islandora Drupal 
+image.
 
-If you haven't already [made a local environment](../docker-local), you'll want to do that first.  These instructions build off of having
-a codebase folder.
+## Creating your Image
 
-## Using your Domain
+In order to generate that custom Drupal image we need to [set up a development environment](../docker-local). You will do this 
+on your development computer, rather than your production server.
 
-At this point, we're assuming that you've purchased a domain to use for your repository.  By default, `islandora.traefik.me` is used, which
-is fine for `make demo` and `make local`.  But for your production site, you'll need to set the domain you own in your .env file.
+Once your development site is set up you will need somewhere to store your custom Drupal image. You should create a private 
+repository in your container registry of choice (Dockerhub, GitHub, GitLab, etc.)
 
+Once you have a place to store it, you can create your custom Drupal image by editing your `.env` to set `CUSTOM_IMAGE_NAMESPACE` 
+to your dockerhub username or the URL and username for your container repository, and `CUSTOM_IMAGE_NAME` to the name of the 
+repository you just created.
+
+Once this is done you can run `make build` to create the image from a Dockerfile. If you don't have a custom Dockerfile, it will 
+create one from sample.Dockerfile. This will create a new image from the Islandora Drupal image, with your codebase folder copied 
+into the image during the build process. 
+
+You should now have a custom Drupal image on your local machine. You will need to `docker login` to your container registry to push 
+your image, so make sure to do that if you haven't yet (you should only need to do this once). You can then push this image to your 
+container registry by running `make push-image`. 
+
+## Set up a Git Repository
+
+Now that you have a development site set up with your own codebase folder, you should create a new git repository for your site. 
+This way you can easily spin up a new site based on your modules and configuration, instead of the Islandora Starter Site. It will 
+also allow you to sync changes between your production, staging, and development sites.
+
+You will likely want to include the Isle-dc directory as well as your codebase folder. This will allow you to make modifications to 
```

Comment:
This part is confusing me a little bit. Is the codebase github repository separate from the isle-dc one or are you committing them into one repository?